src/libs/actions/connections/SageIntacct.ts

```diff
@@ -223,7 +235,14 @@ function updateSageIntacctSyncTaxConfiguration(policyID: string, enabled: boolea
     API.write(WRITE_COMMANDS.UPDATE_SAGE_INTACCT_SYNC_TAX_CONFIGURATION, {policyID, enabled}, {optimisticData, failureData, successData});
 }
 
-function prepareOnyxDataForUserDimensionUpdate(policyID: string, dimensionName: string, newDimensions: SageIntacctDimension[]) {
+function prepareOnyxDataForUserDimensionUpdate(
+    policyID: string,
+    dimensionName: string,
+    newDimensions: SageIntacctDimension[],
+    pendingAction: OnyxCommon.PendingAction = CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+    oldDimensions?: SageIntacctDimension[],
+    oldDimensionName?: string,
```

Comment:
Let's require passing `oldDimensions` and `oldDimensionName` props. Is there a situation when they are not passed?

Reply:
Actually how about always requiring oldValues? I think we should pass them each time